open_seq2seq/losses/centaur_loss.py

```diff
@@ -0,0 +1,188 @@
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+
+import tensorflow as tf
+
+from .loss import Loss
+
+
+class CentaurLoss(Loss):
```

Comment:
Is there a way to merge CentaurLoss and TacotronLoss since they are similar? Maybe a base class that they both inherit from?

Reply:
Merged.